operators/advanced_network/advanced_network/managers/rivermax/rmax_service/ipo_receiver_io_node.cpp

```diff
@@ -1,13 +1,18 @@
 /*
- * Copyright (c) 2017-2024 NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
```

Comment:
the main branch's license looks like a typo `ipo_receiver_io_node.cpp` comparing other files in the same folder

Reply:
Could move to a separate PR if you'd rather avoid blocking these changes on ANO feedback